tests/pages/services/ServicePodReviewScreen-cy.js

```diff
@@ -0,0 +1,1446 @@
+require("../../_support/utils/ServicesUtil");
+
+describe("Services", function() {
+  /**
+   * Test the pods
+   */
+  describe("Pods", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        jobDetails: true,
+        mesos: "1-for-each-health",
+        nodeHealth: true
+      });
+      cy.visitUrl({
+        url: `services/overview/create`
+      });
+    });
+
+    it("renders proper review screen and JSON for a simple pod", function() {
+      const serviceName = "pod-with-inline-shell-script";
+      const command = "while true ; do echo 'test' ; sleep 100 ;";
+
+      cy.contains("Multi-container (Pod)").click();
+
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .type(`{selectall}{rightarrow}${serviceName}`);
+
+      cy.get(".menu-tabbed-item").contains("container-1").click();
+
+      // TODO: Due to a bug in cypress you cannot type values with dots
```

Comment:
what is the `TODO` though? or is it supposed to be only a comment?

Reply:
I would understand it as a "Wait for cypress upgrade and then do", am I right?